src/valid/checks.cpp

```diff
@@ -88,14 +88,15 @@ bool extentsMatchRefs::operator()(const DataArray &extents) const {
 
 bool extentsMatchRefs::operator()(const std::vector<double> &extents) const {
     bool mismatch = false;
+    /*
     auto extSize = extents.size();
     auto it = refs.begin();
     while (!mismatch && (it != refs.end())) {
         auto arrayExtent = (*it).dataExtent();
         mismatch = extSize != arrayExtent.size();
         ++it;
     }
-
+    */
```

Comment:
Saw this in the original PR too.  Do you want to keep the old code around for some reason?

Reply:
yes, thought I want to keep it for reference. Thought one might want to have a validity check of some sort...

Reply:
I am not against keeping it in a comment, but then we should add a comment why it is there.